src/config.js

```diff
@@ -91,4 +94,23 @@ function configToOptions(config = {}) {
   }
 }
 
-export default configToOptions
+const explorer = cosmiconfig('svgr', {
+  transform: result => result && result.config,
+})
+
+async function resolveConfig(opts = {}) {
+  const filePath = resolveConfigFile()
+  const config = await explorer.search(filePath)
+  return configToOptions(merge({}, config, opts))
+}
+
+function resolveConfigFile() {
+  // TODO: not sure where or if we should specify another path;
```

Comment:
@neoziro this is the only thing I'm unsure about right now.

Reply:
I would like to be able to specify a custom path using `--config` and also in command line? Could you add it? Just take inspiration from Prettier.